lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -889,15 +854,17 @@ def support_files_group
             project.pod_support_files_group(pod_name, dir)
           end
 
-          def test_native_target_from_spec_consumer(spec_consumer, test_native_targets)
+          def test_native_target_from_spec(spec, test_native_targets)
```

Comment:
funnily I think this method already exists in `target_installation_result`. and the one below

Reply:
yes it does, I noticed that as well. since this _produces_ the installation result it can't really be reused though